shopgate/pwa
# Hide missing user id error from app user

# Description

Due to some recent changes, currently some users would see an error pop-up in the app. We change the error handling so this is not displayed to the users anymore.

## Type of change


## How to test it

You could throw the EINVALIDCALL error from the connect-trusted extension and still see it in the logs.
